doc/run/repl.md

```diff
@@ -48,7 +48,6 @@ LABEL class
 INTO sqlflow_models.my_dnn_model;
```

Comment:
Maybe this SQL statement is kind of outdated. We need to write:

```sql
SELECT * FROM iris.train
TO TRAIN DNNClassifier
WITH model.n_classes=3, model.hidden_units=[128,64],
    validation.select="SELECT * FROM iris.test"
LABEL class
INTO sqlflow_models.my_dnn_model;
```

now.

Reply:
Thanks for the reminder. I updated the document.

Reply:
thanks @typhoonzero for the comments and @wangkuiyi for the help of resolving the comment!